libs/sdk-ui-kit/src/Dialog/ConversionTouchPointDialogBase.tsx

```diff
@@ -0,0 +1,130 @@
+// (C) 2021 GoodData Corporation
+import React, { useEffect, useState } from "react";
+import { useIntl } from "react-intl";
+import { useHubspotForm } from "@aaronhayes/react-use-hubspot-form";
+import { LoadingComponent } from "@gooddata/sdk-ui";
+
+import { Button } from "../Button";
+import { HubSpotFormLocale } from "@aaronhayes/react-use-hubspot-form/build/main/hubspot-form-locale.enum";
+
+declare global {
+    interface Window {
+        jQuery: object;
+        hbspt: undefined | { isSuccessMessageShow: boolean };
+    }
+}
+
+/**
+ * @internal
+ */
+export interface HubspotFormField {
+    name: string;
+    value: string | any;
+    click: () => void;
+}
+
+/**
+ * @public
+ */
+export interface IConversionTouchPointDialogBaseProps {
+    selectedValue?: string;
+    userEmail: string;
+    clientId: string | number;
+    dialogTitle: string;
+    cancelButtonText?: string;
+    showCancelButton?: boolean;
+    onClose: () => void;
+    onFormSubmitted: () => void;
```

Comment:
Should we marke `onFormSubmitted` as optional? I mean if users don't have any special logic to be executed when the form has been submitted, we should not force them to pass some no-op callback to this field, should we?

※ If we mark this field as optional, please don't forget to change line 80 to `onFormSubmitted?.()` as well.

Reply:
ok, I have updated this. Thanks!